pyproject.toml

```diff
@@ -52,174 +43,159 @@ classifiers = [
-anyio = ">=3"
-attrs = { version = "*", optional = true }
-brotli = { version = "*", optional = true }
-click = "*"
-cryptography = { version = "*", optional = true }
-fast-query-parsers = {version = ">=1.0.2", optional = true }
-httpx = ">=0.22"
-importlib-metadata = { version = "*", python = "<3.10" }
-importlib-resources = { version = ">=5.12.0", python = "<3.9" }
-jinja2 = { version = ">=3.1.2", optional = true }
-jsbeautifier = { version = "*", optional = true }
-mako = { version = ">=1.2.4", optional = true }
-minijinja = {version =">=1.0.0", optional = true}
-msgspec = ">=0.18.2"
-multidict = ">=6.0.2"
-opentelemetry-instrumentation-asgi = { version = "*", optional = true }
-piccolo = { version = "*", optional = true }
-picologging = { version = "*", optional = true }
-polyfactory = ">=2.6.3"
-prometheus-client = { version = "*", optional = true }
-pydantic = { version = "*", optional = true, extras = ["email"] }
-pydantic-extra-types = { version = "*", optional = true }
-python-jose = { version = "*", optional = true }
-pyyaml = "*"
-redis = { version = ">=4.4.4", optional = true, extras = ["hiredis"] }
-rich = ">=13.0.0"
-rich-click = "*"
-sqlalchemy = { version = ">=2.0.12", optional = true }
-structlog = { version = "*", optional = true }
-advanced-alchemy = { version = ">=0.2.2", optional = true }
-typing-extensions = "*"
-uvicorn = { extras = ["standard"], version = ">=0.22.0", optional = true }
-
-[tool.poetry.group.dev.dependencies]
-aiosqlite = "*"
-asyncmy = "*"
-asyncpg = "*"
-beanie = ">=1.21.0"
-beautifulsoup4 = "*"
-duckdb-engine = "*"
-exceptiongroup = { version = "*", python = "<3.11" }
-fsspec = "*"
-greenlet = "*"
-httpx-sse = "*"
-hypothesis = "*"
-opentelemetry-sdk = "*"
-oracledb = "*"
-psycopg = "*"
-pytest = "*"
-pytest-asyncio = "*"
-pytest-cov = "*"
-pytest-lazy-fixture = "*"
-pytest-mock = "*"
-pytest-rerunfailures = "*"
-pytest-timeout = "*"
-pytest-xdist = "*"
-python-dotenv = "*"
-sourcery = "*"
-sqlalchemy-spanner = "*"
-starlette = "*"
-time-machine = "*"
-trio = "*"
-
-[tool.poetry.group.docs]
-optional = true
-
-[tool.poetry.group.docs.dependencies]
-auto-pytabs = { extras = ["sphinx"], version = "*" }
-black = "*"
-sphinx = ">=5.3.0,<=6"
-sphinx-autobuild = "*"
-sphinx-copybutton = ">=0.5.1"
-sphinx-design = ">=0.3.0,<1"
-sphinxcontrib-mermaid = ">=0.8.1,<1"
-litestar-sphinx-theme = { git = "https://github.com/litestar-org/litestar-sphinx-theme.git" }
-sphinx-click = ">=4.4.0"
-
-[tool.poetry.group.lint]
-optional = true
-
-[tool.poetry.group.lint.dependencies]
-asyncpg-stubs = "*"
-black = "*"
-blacken-docs = "*"
-mypy = "*"
-pre-commit = "*"
-pyright = "*"
-ruff = '*'
-slotscheck = "*"
-sourcery = "*"
-types-beautifulsoup4 = "*"
-types-python-jose = "*"
-types-pyyaml = "*"
-types-redis = "*"
-
-[tool.poetry.extras]
+[project.optional-dependencies]
 annotated-types = ["annotated-types"]
 attrs = ["attrs"]
 brotli = ["brotli"]
-cli = ["jsbeautifier", "uvicorn"]
+cli = [
```

Comment:
It looks like the CLI is now included by default?  If we do this, do we even need this group?

Reply:
back compat, i think until 3.0

Reply:
Yes, that was the reason. People might be installing `litestar[cli]`, so if we just remove that, it would break things.